File: Bot/examples/License.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
	"strings"
)

func main() {
	url := "http://<your_ip>:<port>/api/client"
	apiKey := "API_KEY"
	license := "<LICENSE>"
	product := "<PRODUCT>"
	version := "<VERSION>"

	data := strings.NewReader(fmt.Sprintf("license=%s&product=%s&version=%s", license, product, version))

	req, err := http.NewRequest("POST", url, data)
	if err != nil {
		panic(err)
	}

	req.Header.Add("Authorization", apiKey)
	req.Header.Add("Content-Type", "application/x-www-form-urlencoded")

	client := &http.Client{}
	resp, err := client.Do(req)
	if err != nil {
		panic(err)
	}
	defer resp.Body.Close()

	body, err := ioutil.ReadAll(resp.Body)
	if err != nil {
		panic(err)
	}

	fmt.Println(string(body))
}

File: Bot/examples/License.cs
using System;
using System.Net;
using System.IO;

namespace CurlExample
{
    class Program
    {
        static void Main(string[] args)
        {
            string url = "https://uxl.shadowbyte.dev/api/client";
            string apiKey = "XWfQhCTXMhv53622Jt3P7Ae7jiSRrneV5jmcm8E2Quk62j5H";
            string license = "<LICENSE>";
            string product = "Shadow Byte ToolKit";
            string version = "8.0.0";

            string postData = "license=" + license + "&product=" + product + "&version=" + version;

            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
            request.Method = "POST";
            request.Headers.Add("Authorization", apiKey);

            byte[] data = System.Text.Encoding.UTF8.GetBytes(postData);
            request.ContentType = "application/x-www-form-urlencoded";
            request.ContentLength = data.Length;

            Stream stream = request.GetRequestStream();
            stream.Write(data, 0, data.Length);
            stream.Close();

            HttpWebResponse response = (HttpWebResponse)request.GetResponse();
            string responseString = new StreamReader(response.GetResponseStream()).ReadToEnd();
            Console.WriteLine(responseString);
        }
    }
}
